lib/ps.js

```diff
@@ -93,21 +94,32 @@ function ps (pids, options, done) {
       }
 
       var pid = parseInt(line[1], 10)
+
+      var hst = history.get(pid, options.maxage)
+      if (hst === undefined) hst = {}
+
       var ppid = parseInt(line[2], 10)
-      var cpu = parseFloat(line[3].replace(',', '.'), 10)
       var memory = parseInt(line[4], 10) * 1024
       var etime = parseTime(line[0])
       var ctime = parseTime(line[5], true)
 
+      var total = (ctime - (hst.ctime || 0))
+      // time elapsed between calls in seconds
+      var seconds = Math.abs(hst.etime !== undefined ? etime - hst.etime : etime)
+      var cpu = seconds > 0 ? (total / seconds) * 100 : 0
```

Comment:
lmk what you think about this @simonepri

Reply:
I would use the same structure used in wmic:
https://github.com/soyuka/pidusage/blob/master/lib/wmic.js#L120